VideoRenderer/VideoRenderer/PictureInPictureControllerObserver.swift

```diff
@@ -0,0 +1,59 @@
+//  Copyright © 2017 One by Aol : Publishers. All rights reserved.
+
+import Foundation
+import AVKit
+
+public final class PictureInPictureControllerObserver: NSObject {
```

Comment:
`PlayerObserver` is a bit outdated - I would like to migrate to `SystemPlayerObserver` in nearest future.